src/types.tsx

```diff
@@ -218,7 +217,9 @@ export interface ScreenProps extends ViewProps {
   /**
    * A callback that gets called after swipe back is canceled.
    */
-  onGestureCancel?: (e: NativeSyntheticEvent<null>) => void;
+  onGestureCancel?: (
+    e: NativeSyntheticEvent<Readonly<Record<string, never>>>,
+  ) => void;
```

Comment:
Why do we change this?

Reply:
Yeah, nothing changed here

Reply:
Reverted in: https://github.com/software-mansion/react-native-screens/pull/2423/commits/c51f636ea6d26c8f2752d839cc8452e3874a48e7